File: lib/layer2_core/dark/e_secondary_fixed_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class ESecondaryFixedDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd29;
      case 'standard': return cstd29;
      case 'blue': return cbd29;
      case 'green': return cgd29;
      case 'red': return crd29;
      case 'nothing': return cnod29;
      default: return cstd29;
    }
  }
}

File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/ESecondaryFixedDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object ESecondaryFixedDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd29
            AppTheme.STANDARD -> R.color.cstd29
            AppTheme.BLUE -> R.color.cbd29
            AppTheme.GREEN -> R.color.cgd29
            AppTheme.RED -> R.color.crd29
            AppTheme.NOTHING -> R.color.cnod29
        }
    }
}
